services/auth/source/ldap/source_search.go

```diff
@@ -117,10 +117,11 @@ func dial(source *Source) (*ldap.Conn, error) {
 	}
 
 	if source.SecurityProtocol == SecurityProtocolLDAPS {
-		return ldap.DialTLS("tcp", net.JoinHostPort(source.Host, strconv.Itoa(source.Port)), tlsConfig)
+		dialOpt := ldap.DialWithTLSConfig(tlsConfig)
+		return ldap.DialURL(net.JoinHostPort(source.Host, strconv.Itoa(source.Port)), dialOpt)
```

Comment:
I think you need to prefix with `ldaps://` as per https://pkg.go.dev/gopkg.in/go-ldap/ldap.v3#DialURL